installation/install.sh

```
@@ -2241,16 +2247,21 @@ uz_upgrade_zstack(){
     else
-        bash zstack/WEB-INF/classes/tools/install.sh zstack-cli >>$ZSTACK_INSTALL_LOG 2>&1
+        bash cloud/WEB-INF/classes/tools/install.sh zstack-cli >>$ZSTACK_INSTALL_LOG 2>&1
     fi
     if [ $? -ne 0 ];then
         cd /; rm -rf $upgrade_folder
         fail "failed to upgrade ${PRODUCT_NAME,,}-cli"
     fi
 
-    bash ${zstore_bin} >>$ZSTACK_INSTALL_LOG 2>&1
+    # Execute zstore_bin from the temporary extracted cloud directory
+    if [ `uname -m` == "x86_64" ]; then
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)